Makefile

```diff
@@ -26,8 +23,12 @@ version.properties:
 .target_os:
 	@echo $(OS) > .target_os
 
-$(TARGET): .target_os $(SOURCES)
-	GOOS=$(OS) GOARCH=${ARCH} CGO_ENABLED=0 go build -a -installsuffix cgo --ldflags '-w' -o $@ main.go
+build: $(TARGET)
+
+$(TARGET): .target_os
```

Comment:
If you remove `$(SOURCES)`  from the target dependencies, will it rebuild if you modify a source file?

Reply:
Since I added `$(TARGET)` to `.PHONY` it will always trigger